operations/reservations.md

```diff
@@ -1006,22 +922,25 @@ Updates information about the specified reservations. Note that if any of the fi
 
 Same structure as in [Get all reservations](#get-all-reservations) operation.
```

Comment:
Something I just noticed, nothing to do with the re-ordering of the operations - this line should reference the specific version or versions of `Get all reservations`, otherwise it is ambiguous.

Reply:
Good point, but this paragraph is not present in the generated documentation – we just repeat the same response again.